django/db/models/sql/compiler.py

```diff
@@ -1504,7 +1505,7 @@ def as_sql(self):
             pk.get_col(self.query.get_initial_alias())
         ]
         outerq = Query(self.query.model)
-        outerq.where = self.query.where_class()
+        outerq.where = WhereNode()
```

Comment:
Is this redundant with `Query.__init__()`?

Reply:
Good shout. Before it needed to pass through the same type of node that was used by the (now) inner query.